test/workbox-google-analytics/integration/test-all.js

```diff
@@ -96,22 +92,32 @@ describe(`[workbox-google-analytics] initialize`, function() {
       action: 'clear-spied-requests',
     });
 
-    // Check the "simulate offline" checkbox and make some requests.
-    await simulateOfflineEl.click();
+    // Switch the service worker into "offline" mode.
+    await webdriver.executeAsyncScript(messageSW, {
+      action: 'simulate-offline',
+      value: true,
+    });
+
     await webdriver.executeAsyncScript((done) => {
       window.gtag('event', 'beacon', {
         transport_type: 'beacon',
         event_label: Date.now(),
-        event_callback: () => done(),
+        event_callback: () => {
+          setTimeout(done, 50);
+        },
       });
     });
+
     await webdriver.executeAsyncScript((done) => {
       window.gtag('event', 'pixel', {
         transport_type: 'image',
         event_label: Date.now(),
-        event_callback: () => done(),
+        event_callback: () => {
+          setTimeout(done, 50);
```

Comment:
Can you add a comment that links to https://github.com/GoogleChrome/workbox/issues/2168 here?